recipes/console_bridge/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm
+
+required_conan_version = ">=1.53.0"
+
+
+class PackageConan(ConanFile):
+    name = "console_bridge"
+    description = "A ROS-independent library for logging that seamlessly pipes into rosconsole/rosout for ROS-dependent packages"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ros/console_bridge"
+    topics = ("logging", "ros", "robotics")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 14
+
+    @property
+    def _tests_enabled(self):
+        return not self.conf.get("tools.build:skip_test", default=True, check_type=bool)
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        pass
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+
+    def build_requirements(self):
+        if self._tests_enabled:
+            self.build_requires("gtest/1.13.0")
```

Comment:
please, do not include unit tests in CCI recipes. Let's try to keep it simpler, as the focus is packaging only, not checking project features. Once you add this support, anyone can open an issue and say some unit test is broken, so it will be configured as a bug. 
```suggestion
            self.build_requires("gtest/1.13.0")
```